components/Banner/Banner.tsx

```diff
@@ -51,7 +51,7 @@ export const Banner = ({
       </Container>
       {imageSrc && (
         <div className={styles.imageWrapper}>
-          <AnimateInView duration={0.4} delay={0.7} animation="slideInFromRight">
+          <AnimateInView delay={0.7} animation="slideInFromRight">
```

Comment:
For example I remove the animation duration that's passed into the Banner content and image here so we go with the default 0.6 in the `AnimateInView` wrapper